testsuite/tests/mgc/test_basic.py

```diff
@@ -0,0 +1,110 @@
+"""
+This module contains the very basic tests and their dependencies for MGC
+"""
+from time import sleep
+import httpx
+import pytest
+from testsuite.openshift.httpbin import Httpbin
+from testsuite.openshift.objects.gateway_api import MGCGateway, HTTPRoute
+
+
+# prerequisites:
```

Comment:
Move this to the module doc string and add a bullet list toa the actual prerequisites